cvm-securekey-release-app/AttestationUtil.h

```diff
@@ -84,6 +96,20 @@ class Util{
         }
     }
 
+    inline static std::string applyMask(const std::string& str)

```

Comment:
the name of the method is a bit generic, since mask is used in various contexts. How about reduct_log(...)

Reply:
agree, I will do this change.

Reply:
done